docs/rollover.md

```diff
@@ -0,0 +1,44 @@
+# Rollover
+
+Each year we close the current cycles courses and open the new cycle in a
+process we call 'Rollover'.
+
+This involves copying existing providers and courses to new records to allow the
+providers to update any details and then switching the API over to a new
+'recruitment cycle' which in turn releases the new courses on Find & Apply.
+
+This document lists the changes needed to be made to the TTAPI codebase and the
+timings for these changes. There is separate documentation for Publish
+[here](https://github.com/DFE-Digital/publish-teacher-training/blob/master/docs/rollover.md).
+
+## Testing the Rollover process
```

Comment:
Might be worth calling out the need to add the `recruitment_cycle` here, even though it's mentioned further down ... it'll be needed to test the script. Something like this maybe:

> ## Preparing for Rollover
>
> Before rollover can happen, a new entry has to be added to the `recruitment_cycle` table for the next cycle. The `application_start_date` and `application_end_date` may not be known yet as those are revealed later in the year, so you can just use sensible placeholders.

Is there anything else that needs to be prepared?

Reply:
I believe that's the only thing needed before running the copying code - updated!